docker-compose.yml

```diff
@@ -9,9 +9,13 @@ services:
     build: ./
     env_file:
       - .env
+    environment:
```

Comment:
I think it's better if we just provide one location for setting the env variables. This is going to catch someone out